### PR TITLE
Jen/unnamed facility

### LINKED ISSUE #124: Create the Empty State Facility Page
When a logged-in user who has not yet added any facilities accesses the Multi-facility page, they should see a page that allows them to add a new facility.

Screen Shot 2020-04-17 at 10 42 32 AM

Full mock can be found here:  https://www.figma.com/file/EGJS9njuKHWdFHUS4lrHA4/%F0%9F%9A%80Recidiviz-CJStatus?node-id=910%3A130

Theoretically, this page should be identical to the current Impact Dashboard with that the user can also name the facility.

When the user clicks 'Save', the Facility should be stored in Cloud Firestore and the user should be redirected to the Multi-facility page dashboard.

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

Adds the UnnamedFacilityPage

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #124

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/page-unnamed-facility/RateOfSpreadSection.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  },
+  incarcerated: {
+    visitation: {
+      value: false,
+      label: "In-person visitation has been suspended",
+    },
+    restrictedRecreactionTime: {
+      value: false,
+      label:
+        "Rectreaction time has been restricted to the same groups of individuals daily, and in small numbers (<12)",
```
Comment (jessex) [inline src/page-unnamed-facility/RateOfSpreadSection.tsx:61]: Whoops: `Recreation`

**Comment by macfarlandian** [comment]
@jovergaag when it comes to the data .... anything in here that was already in the `ImpactDashboard` should work exactly the same, using the `EpidemicModelContext`, as long as your branch is up to date with `master`. (Anything you don't want to spend time wrestling with today, you are welcome to leave it for me!)

Anything that wasn't in `ImpactDashboard` (e.g. the policy checkboxes), don't stress about it because it's probably going to be turned off for the next release anyway — we've basically taken any additional inputs out of scope for that, but I expect to be working on that next week.

**Comment by jovergaag** [comment]
@macfarlandian Thanks for that comment. I took the `FacilityInformation` and `ImpacrtProjectionTable` straight out of the `ImpactDashboard`.  `LocaleInformation` and `RateOfSpread` had new inputs so I tried to customize them a bit.  Should I just scrap that customization work and copy those sections straight from the `ImpactDashboard`? I didn't realize that the checkboxes, etc were not part of this release. Maybe that was mentioned in standup so I missed it...

**Comment by macfarlandian** [comment]
@jovergaag it may have been mentioned at some point but it probably wasn't totally clear; the mocks are unfortunately not very explicit about the state of the model so we have made this kind of mistake more than once ... just an unfortunate side effect of our breakneck speed I think 😐 

customization in terms of adjusting the look and feel is probably fine as long as it doesn't take too long; customization in terms of adding new input fields is not worth it at this point but anything done or partially done can be parked somewhere to get picked up next week, it's certainly not throwaway work! On the other hand if simply reusing the existing components and making the visual adjustments directly to them works, it's fine to do that too. whatever gets us there IMO

**Comment by jovergaag** [comment]
@macfarlandian @jessex I think this is in a pretty good place to merge.  Do you want to take a quick look?

**Pull request merged**